doc/fileformat/cable_cell.rst

```diff
@@ -0,0 +1,440 @@
+.. _formatcablecell:
+
+Arbor Cable Cell format (ACC)
+=============================
+
+We define an s-expression format for describing :ref:`cable cells <cablecell>`.
+Cable cells are constructed from three components: a :ref:`label dictionary <labels>`,
+a :ref:`decoration object <cablecell-decoration>` and a :ref:`morphology <morph>`.
+The cable cell *format* is constructed in the same way.
+
+.. Note::
+
+   Lines breaks and indentation in the s-expressions presented below have been added
+   for clarity. They are not a requirement of the format and will be ignored by the
```

Comment:
It's probably more precise to say 'treated as whitespace', but I unsure whether this matters, eg
```LISP
(loc
  ation 0 0)
```
will be parsed differently than
```LISP
(location 0 0)
```

Reply:
Yeah, the first will fail. I mean indentations and white space between symbols. I can say 'treated as whitespace', will that be enough?

Reply:
Yes, as I said 'Treated as whitespace' is more precise, but whether the reader cares (or sees the equivalence directly) I do not know.

Reply:
"Extra line breaks and indentations in the s-expressions presented below have been added for clarity. They are not a requirement of the format and will be treated as whitespace."?

Reply:
👍